rules/__tests__/valid-describe.test.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rules = require('../..').rules;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 8,
+  },
+});
+
+ruleTester.run('valid-describe', rules['valid-describe'], {
+  valid: [
+    'describe("foo")',
+    'describe("foo", function() {})',
+    'describe("foo", () => {})',
+    'xdescribe("foo", () => {})',
+    'fdescribe("foo", () => {})',
+    'describe.only("foo", () => {})',
+    'describe.skip("foo", () => {})',
+    `
+    describe('foo', () => {
+      it('bar', () => {
+        return Promise.resolve(42).then(value => {
+          expect(value).toBe(42)
+        })
+      })
+    })
+    `,
+    `
+    describe('foo', () => {
+      it('bar', async () => {
+        expect(await Promise.resolve(42)).toBe(42)
+      })
+    })
+    `,
+  ],
+  invalid: [
+    {
+      code: 'describe("foo", async () => {})',
```

Comment:
as a follow up, mind adding a rule that `describe(() => {})` is invalid as well?

Reply:
For sure, I'll open an issue to track that suggestion. 👍